packages/pixeloven-cli/src/index.test.ts

```diff
@@ -8,13 +14,38 @@ const caller = () => {
 describe("@pixeloven/cli", () => {
     describe("index", () => {
         afterAll(() => {
-            jest.clearAllMocks();
-            jest.restoreAllMocks();
+            sandbox.restore();
+        });
+        afterEach(() => {
+            sandbox.reset();
         });
         it("should execute main and succeed", () => {
-            const cliSpy = jest.spyOn(cli, "default");
+            mainSpy.expects("default").once();
             caller();
-            expect(cliSpy).toHaveBeenCalledTimes(1);
+            mainSpy.verify();
         });
     });
 });
+
+// import "jest";
```

Comment:
Looks like this is all duplication, can it be removed?

Reply:
This isn't ready for PR haha! I usually create a PR with no reviewers so I can track my changes more easily.

Reply:
whoops, sorry. Lemme know when you want a second look at it.